pkg/importer/spanner/test-spanner.sql

```diff
@@ -0,0 +1,37 @@
+CREATE DATABASE CustomerAccounts;
```

Comment:
Nit: Can you add a comment at the top of this file, partly to describe what it is, and partly to test that the grammar can handle comments?

Reply:
ok